docs/source/getstarted.rst

```diff
@@ -27,12 +27,13 @@ Please visit our `Cozmo SDK Forums <https://forums.anki.com/>`_ where you can:
 Starting Up the SDK
 -------------------
 
-1. Plug the mobile device containing the Cozmo app into your computer.
-2. Open the Cozmo app on the phone. Make sure Cozmo is on and connected to the app via WiFi.
-3. Tap on the gear icon at the top right corner to open the Settings menu.
-4. Swipe left to show the Cozmo SDK option and tap the Enable SDK button.
-5. On the computer, open a Terminal (macOS/Linux) / Command Prompt (Windows) window. Type ``cd cozmo_sdk_examples`` where *cozmo_sdk_examples* is the name of the directory you extracted the SDK examples into and press Enter. The SDK example scripts can be downloaded from the :doc:`Downloads page </downloads>`.
-6. Now you can run any of the example programs.
+1.	Plug the mobile device containing the Cozmo app into your computer.
+2.	Open the Cozmo app on the phone. Make sure Cozmo is on and connected to the app via WiFi.
+3.	Tap on the gear icon at the top right corner to open the Settings menu.
+4.	Swipe left to show the Cozmo SDK option and tap the Enable SDK button.
+5.	Make sure the SDK examples are downloaded from the :doc:`Downloads page </downloads>`.  Remove the version number from the folder name so that it is just called “cozmo_sdk_examples”.
+6.	To run a program, open Terminal (macOS/Linux) or Command Prompt (Windows) and navigate to the folder containing that program. For example, if the cozmo_sdk_examples folder is saved to your desktop, type ``cd Desktop/cozmo_sdk_examples/tutorials/01_basics``
```

Comment:
I like the old "On the computer" text that began this step. It clarifies that the user is not on the device for this bit.

Also, I think it makes sense to have the user to to the cozmo_sdk_examples folder and not yet drill down into 01_basics. In "Hello World" we tell them to cd into 01_basics at that point.